samples/web/content/manual-test/multiple-video/index.html

```diff
@@ -68,5 +47,25 @@
           autoplay="autoplay"></video></td>
     </tr>
   </table>
+  <script src="../../../js/adapter.js"></script>
```

Comment:
Due to Sam's comment in https://github.com/GoogleChrome/webrtc/pull/277 (first outdated comment).